DQMOffline/L1Trigger/python/L1TEGammaEfficiency_cfi.py

```diff
@@ -81,28 +40,33 @@
     'photon': [],
 }
 
-allEfficiencyPlots_HI = []
-add_plot = allEfficiencyPlots_HI.append
-for variable, thresholds in variables_HI.iteritems():
-    for plot in plots[variable]:
-        for threshold in thresholds:
-            plotName = '{0}_threshold_{1}'.format(plot, threshold)
-            add_plot(plotName)
+# remove photon variables (code to produce them is currently commented out)
+variables['photon'] = []
+variables_HI['photon'] = []
 
-for variable, thresholds in deepInspectionThresholds_HI.iteritems():
-    for plot in deepInspectionPlots[variable]:
-        for threshold in thresholds:
-            plotName = '{0}_threshold_{1}'.format(plot, threshold)
-            add_plot(plotName)
+from DQMOffline.L1Trigger.L1TCommon import generateEfficiencyStrings as ges
+efficiencyStrings = list(ges(variables, plots))
+efficiencyStrings += list(ges(deepInspectionThresholds, deepInspectionPlots))
 
-from Configuration.Eras.Modifier_ppRef_2017_cff import ppRef_2017
-ppRef_2017.toModify(l1tEGammaEfficiency,
-    plotCfgs = {
-        0:dict(plots = allEfficiencyPlots_HI),
-    }
+efficiencyStrings_HI = list(ges(variables_HI, plots))
+efficiencyStrings_HI += list(ges(deepInspectionThresholds_HI,
+                                 deepInspectionPlots))
+
+from DQMServices.Core.DQMEDHarvester import DQMEDHarvester
+l1tEGammaEfficiency = DQMEDHarvester(
+    "DQMGenericClient",
+    commands=cms.vstring(),
```

Comment:
`commands` does not seem to be a parameter of DQMGenericClient.

Reply:
Maybe not any more?
https://github.com/cms-sw/cmssw/blob/master/HLTriggerOffline/Common/python/PostProcessorExample_cfi.py

Reply:
In that case for a long time.